README.md

```diff
@@ -24,6 +24,24 @@ GRAYLOG_PATH - Graylog server input path (default: '/gelf')
 GRAYLOG_SOURCE - Graylog name for the logger Host
 ```
 
+# Local Development
+For local devolpment we use dev containers since libreoffice features that we use for document conversion is very linux dependent.
+You can find dev container definition on .devcontainer folder.
+
+To prepare local development
+1. Initiate the redis container that it is defined on the docker-compose file

```

Comment:
To make it simpler, could we integrate a docker-compose with the dev container so that it will start redis along with the dev container? Maybe also use a non-standard port for redis so that it doesn't interfere with another redis instance that might already be running

Reply:
I refactored to make it more simpler and docker-composer based